### PR TITLE
AutoCorrect files/zh-cn/web/api/{w,x}*/

### PR DESCRIPTION
**Pull request by huacnlee**

### CODE REVIEW

File: files/zh-cn/web/api/xmldocument/async/index.html [hunk 1]
```diff
@@ -5,15 +5,15 @@
 ---
 <p>{{APIRef("DOM")}}{{Non-standard_header}}{{Deprecated_header}}</p>
 
-<p><code>document.async</code> 可以被赋值,用来表明 <code><a href="/zh-cn/DOM/document.load" title="zh-cn/DOM/document.load">document.load</a></code> 被调用时,是使用异步模式还是同步模式进行请求. <code>true</code> 为默认值, 表明该文档应该被异步加载.</p>
+<p><code>document.async</code> 可以被赋值，用来表明 <code><a href="/zh-cn/DOM/document.load" title="zh-cn/DOM/document.load">document.load</a></code> 被调用时，是使用异步模式还是同步模式进行请求。<code>true</code> 为默认值，表明该文档应该被异步加载。</p>
 
-<p>(从Gecko 1.4alpha 开始,文档请求可以以异步模式进行,在这之前,只有同步模式可用.)</p>
+<p>(从 Gecko 1.4alpha 开始，文档请求可以以异步模式进行，在这之前，只有同步模式可用.)</p>
```
Comment (yin1999) [inline files/zh-cn/web/api/xmldocument/async/index.html:10]: seems not take effect:

```suggestion
<p>(从 Gecko 1.4alpha 开始，文档请求可以以异步模式进行，在这之前，只有同步模式可用。)</p>
```

File: files/zh-cn/web/api/xmldocument/load/index.html [hunk 2]
```diff
@@ -5,19 +5,19 @@
 ---
 <p>{{APIRef("DOM")}}</p>
 
-<p><code>document.load()</code>作为旧版的w3c标准 <a class="external" href="http://www.w3.org/TR/2003/WD-DOM-Level-3-LS-20030619/load-save.html#LS-DocumentLS">DOM Level 3 Load &amp; Save module</a> 其中的一部分. <a href="/zh-cn/DOM/document.async" title="zh-cn/DOM/document.async">document.async</a> 用来表明该请求是以同步模式进行还是异步模式进行(默认值). 从 Gecko 1.9开始 ,该方法不支持跨站的文档请求.(使用 <a href="/zh-cn/DOM/XMLHttpRequest" title="zh-cn/XMLHttpRequest">XMLHttpRequest</a> 代替).</p>
+<p><code>document.load()</code>作为旧版的 w3c 标准 <a class="external" href="http://www.w3.org/TR/2003/WD-DOM-Level-3-LS-20030619/load-save.html#LS-DocumentLS">DOM Level 3 Load &amp; Save module</a> 其中的一部分。<a href="/zh-cn/DOM/document.async" title="zh-cn/DOM/document.async">document.async</a> 用来表明该请求是以同步模式进行还是异步模式进行 (默认值). 从 Gecko 1.9 开始 ,该方法不支持跨站的文档请求.(使用 <a href="/zh-cn/DOM/XMLHttpRequest" title="zh-cn/XMLHttpRequest">XMLHttpRequest</a> 代替).</p>
```
Comment (yin1999) [inline files/zh-cn/web/api/xmldocument/load/index.html:8]: and this one

**Review by yin1999 — COMMENTED** [review]
I'm not sure about this

**Review by huacnlee — COMMENTED** [review]
18 个文件已检查

File: files/zh-cn/web/api/waveshapernode/index.html [hunk 3]
```diff
@@ -6,7 +6,7 @@
 <p>{{ APIRef("Web Audio API") }}</p>
 
 <div>
-<p><strong><code>WaveShaperNode</code></strong> 接口表示一个非线性的畸变器。 是一个使用曲线来将一个波形畸变应用到一个声音信号中的{{domxref("AudioNode")}} 。 除了明显的失真效果之外, 它通常用来给信号添加一个暖调的感觉。</p>
+<p><strong><code>WaveShaperNode</code></strong> 接口表示一个非线性的畸变器。是一个使用曲线来将一个波形畸变应用到一个声音信号中的{{domxref("AudioNode")}}。 除了明显的失真效果之外，它通常用来给信号添加一个暖调的感觉。</p>
```
Comment (yin1999) [inline files/zh-cn/web/api/waveshapernode/index.html:9]:
```suggestion
<p><strong><code>WaveShaperNode</code></strong> 接口表示一个非线性的畸变器。是一个使用曲线来将一个波形畸变应用到一个声音信号中的{{domxref("AudioNode")}}。除了明显的失真效果之外，它通常用来给信号添加一个暖调的感觉。</p>
```

File: files/zh-cn/web/api/workerglobalscope/index.html [hunk 4]
```diff
@@ -14,9 +14,9 @@
...
+<p> <a href="/en-US/docs/Web/API/Web_Workers_API">Web Workers API</a>  的 <code><strong>WorkerGlobalScope</strong></code> 接口 是一个代表了任何 scope of worker 的接口. Workers 没有浏览内容；这个 scope 包含的信息总是通过 {{domxref("Window")}} objects 传递 — 比如 event handlers, the console or the associated {{domxref("WorkerNavigator")}} object。每个 <code>WorkerGlobalScope</code> 都有自己的事件循环。</p>
 
-<p>每个 worker type都有专门的这个接口: {{domxref("DedicatedWorkerGlobalScope")}} for dedicated workers, {{domxref("SharedWorkerGlobalScope")}} for shared workers, and {{domxref("ServiceWorkerGlobalScope")}} for <a href="/en-US/docs/Web/API/ServiceWorker_API">ServiceWorker</a>. <code>self</code> 属性返回每个内容的专门 scope .</p>
+<p>每个 worker type 都有专门的这个接口：{{domxref("DedicatedWorkerGlobalScope")}} for dedicated workers, {{domxref("SharedWorkerGlobalScope")}} for shared workers, and {{domxref("ServiceWorkerGlobalScope")}} for <a href="/en-US/docs/Web/API/ServiceWorker_API">ServiceWorker</a>. <code>self</code> 属性返回每个内容的专门 scope .</p>
```
Comment (yin1999) [inline files/zh-cn/web/api/workerglobalscope/index.html:19]:
```suggestion
<p>每个 worker type 都有专门的这个接口：{{domxref("DedicatedWorkerGlobalScope")}} for dedicated workers, {{domxref("SharedWorkerGlobalScope")}} for shared workers, and {{domxref("ServiceWorkerGlobalScope")}} for <a href="/en-US/docs/Web/API/ServiceWorker_API">ServiceWorker</a>. <code>self</code> 属性返回每个内容的专门 scope.</p>
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Pull request merged**